nucypher/blockchain/economics.py

```diff
@@ -66,9 +66,9 @@ class BaseEconomics:
 
     # Adjudicator
     _default_hash_algorithm = HASH_ALGORITHM_SHA256
-    _default_base_penalty = 100
-    _default_penalty_history_coefficient = 10
-    _default_percentage_penalty_coefficient = 8
+    _default_base_penalty = 2
+    _default_penalty_history_coefficient = 0
```

Comment:
I think this should remain zero until there is a compelling behavioural and/or threat-based rationale to ramp penalties. Hypothetically, if a single incorrect re-encryption could occur innocently, but five from the same Ursula is no accident (and thus indicates an attack), then it makes sense to considerably weaken that staker with increasing slashes. But in the absence of a good reason, and with community resistance, this component will do more harm than good.

Reply:
I think the argument for low but non-zero numbers is that it gives us a relatively low-risk way to evaluate the slashing logic in production.

Reply:
That argument is completely valid. But 'evaluating' the slashing mechanism involves observing real-world behaviour, not only testing the code. If in production, a group of stakers start repeatedly producing incorrect re-encryptions, then that certainly provides a compelling reason to increase the punishment and/or introduce ramping. As it stands, we can't concretely articulate the benefit of producing incorrect re-encryptions to a staker, nor the risk to the network. This makes it difficult to parametrize the difference in punishment between one incorrect re-encryption and five. However, this may change once the network is live!